esp/esp/users/forms/user_profile.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, *args, **kwargs):
 
     def clean(self):
         super(UserContactForm, self).clean()
-        if self.user.isTeacher() or Tag.getBooleanTag('request_student_phonenum', default=True):
+        if self.user.isTeacher():
```

Comment:
Maybe it would be more appropriate to change the default for the tag?

Reply:
Do we really think students won't have either phone type?

Reply:
I think default true makes sense because we do want to ask for their phone number in most cases? But yes, in New Haven at least, we get many students without cell phones. Not sure about home phones but those are getting rarer and rarer anyways.

Reply:
If you change the default to "False", that means the value will be "False" if no tag is set. If someone wants to request student phone numbers, then they would need to set the tag as "True". As you have it now, the tag does nothing.